src/constants.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+global PROJECT_DIRECTORY
+global LOCAL_DIRECTORY
+
+global EXPERIMENT_ANALYSIS_FILE
+global MEDIA_ANALYSIS_FILE
+global HEADERS_FILE
+global EXPERIMENTS_LIST
+global MEDIA_LIST
+
+global od_regex
+global counts_regex
+global qpcr_regex
+global rnaseq_regex
+
+global abundance_options
+
+# This directory MUST BE the one where the core is in the server
+PROJECT_DIRECTORY = '/Users/julia/bacterialGrowth_thesis/'
+LOCAL_DIRECTORY = '/Users/julia/Desktop/local_thesis_files/'
+# Bash files needed to run the code
+EXPERIMENT_ANALYSIS_FILE = 'src/bash_scripts/Lab_files_analysis2.sh'
+MEDIA_ANALYSIS_FILE = 'src/bash_scripts/media_file_analysis.sh'
+MODIFY_YML_FILE = 'src/bash_scripts/modify_yml_files.sh'
+
+# Files generated by the bash needed in the code
+HEADERS_FILE = 'IntermediateFiles/lab_headers.txt'
+EXPERIMENTS_LIST = 'IntermediateFiles/listOfFiles.list'
+MEDIA_LIST = 'IntermediateFiles/listOfMedia.list'
+
+# File types
+file_types = ['abundanceFile', 'metabolitesFile', 'phFile']
+
+# Abundance options
+abundance_options = ['od', 'counts', 'qpcr', 'rnaseq']
+
+# Regex options
```

Comment:
Let's keep those like this for now and we see later what's the best way to go.

Reply:
Maybe it could be nice to open an issue for this task.
We ll consider of integrating the [MCO ontology](https://www.ebi.ac.uk/ols/ontologies/mco)